aws/s3-bucket/encryption.tf

```diff
@@ -0,0 +1,11 @@
+resource "aws_s3_bucket_server_side_encryption_configuration" "encryption" {
+  count = var.encrypted ? 1 : 0
```

Comment:
QoL: I feel like we should just have the sse encryption enabled by default. Can't you think of any drawbacks?
```suggestion

```

Reply:
you are right, changed it so that it is always encrypted